frontend/public/components/storage/attach-storage.tsx

```diff
@@ -94,7 +95,9 @@ const AttachStorage: React.FC<AttachStorageFormProps> = (props) => {
           </div>
         </>
       )}
-      <AsyncComponent loader={storageProvidersMap[activeProvider].Component} {...props} />
+      <ErrorBoundary FallbackComponent={ErrorBoundaryFallback}>
+        <AsyncComponent loader={storageProvidersMap[activeProvider].Component} {...props} />
+      </ErrorBoundary>
```

Comment:
I might be wrong, but this looks like it should be another 'inline' ErrorBoundary.

Reply:
Not really... the content is very tiny around the error -- I think a full page error works better in this case. 

![Screen Shot 2022-06-15 at 3 09 19 PM](https://user-images.githubusercontent.com/8126518/173905838-40e816e5-dea5-4120-ab3c-62fc44f56d8e.png)
![Screen Shot 2022-06-15 at 3 09 25 PM](https://user-images.githubusercontent.com/8126518/173905842-914bf7aa-e2b8-4cc8-87dd-8bb71b79cf6f.png)